packages/devtools-reps/src/launchpad/index.js

```diff
@@ -33,13 +33,7 @@ function onConnect(connection) {
     },
     getProperties: async function (grip) {
       const objClient = connection.tabConnection.threadClient.pauseGrip(grip);
-
       const resp = await objClient.getPrototypeAndProperties();
-      const { ownProperties, safeGetterValues } = resp;
-      for (const name in safeGetterValues) {
-        const { enumerable, writable, getterValue } = safeGetterValues[name];
-        ownProperties[name] = { enumerable, writable, value: getterValue };
-      }
```

Comment:
awesome - this was always a hack to be able to use getter values without component support for getterValues.